src/utils/cron.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-useless-path-segments */
+/* eslint-disable node/no-missing-require */
+/* eslint-disable import/no-unresolved */
+
+const mongoose = require('mongoose');
+const cron = require('node-cron');
+const sendEmail = require('../utils/mailer');
+const logger = require('../utils/logger');
+const Project = require('../models/project');
+
+const today = new Date();
+
+async function initializeCronTasks() {
+  cron.schedule(
+    '0 9 * * *',
+    () => {
+      const upcomingProjects = async () => {
+        const projects = await Project.find();
+        const titles = [];
+        // eslint-disable-next-line no-plusplus
+        for (let i = 0; i < projects.length; i++) {
+          if (projects[i].date.toString() > today.toString()) {
+            titles.push(projects[i].title);
+          }
+        }
+        sendEmail(
+          'm.cankisi@gmail.com',
+          'Upcoming Voluntera Projects',
+          titles.join()
+        );
+      };
+      upcomingProjects();
+    },
```

Comment:
`sendEmail` requires you to send a html string. You can do the following:
```suggestion
    async () => {
      const date = new Date();
      date.setDate(date.getDate() - 1);

      const projectTitles = await Project.find({
        createdAt: { $gte: date },
      }).select({ titles: 1 });

      const htmlTemplate = `<p>${projectTitles.join('\n')}</p>`;
      // eslint-disable-next-line no-plusplus
      sendEmail(
        'm.cankisi@gmail.com',
        'Upcoming Voluntera Projects',
        htmlTemplate
      );
    },
```

Reply:
Hi Eniz
for the first changes request I tried it but it gave a lot of problems so I will keep the eslint-disable for now.
and I'm applying the HTML function here as you suggested. Thanks!

Reply:
Hey Can, don't forget you still need to install node-cron. It's not in the package.json